src/app/directives.js

```diff
@@ -12,4 +12,26 @@ angular.module('librarium.directives', [])
         });
       }
     };
-  }]);
+  }])
+  .directive('starRating', function () {
```

Comment:
instead of creating your own directive you can use https://angular-ui.github.io/bootstrap/#!#rating It's always a better idea to reuse libraries that are already tested, unless you can't find what you need anywhere. Don't reinvent the wheel